manual/add-note.php

```diff
@@ -367,7 +367,7 @@
   </tr>
   <tr>
    <th class="subr"><label for="form-user">Your email address (or name)</label>:</th>
```

Comment:
```suggestion
   <th class="subr"><label for="form-user">Your email address</label>:</th>
```

I suppose we could additionally do that?